packages/oui-field/README.md

```diff
@@ -149,7 +160,20 @@
 
 ### oui-datagrid
 
-| Attribute         | Type            | Binding | One-time binding | Values                    | Default             | Description                                       |
-| ----              | ----            | ----    | ----             | ----                      | ----                | ----                                              |
-| `helper-text`     | string          | @?      | yes              |                           |                     | the field label                                   |
-| `label`           | function        | @?      | yes              |                           |                     | a text to help fill the form field                |
+| Attribute         | Type            | Binding | One-time binding | Values      | Default             | Description                                       |
+| ----              | ----            | ----    | ----             | ----        | ----                | ----                                              |
+| `error-messages`  | object          | <?      | no               |             |                     | a dictionary to override default messages         |
+| `helper-text`     | string          | @?      | yes              |             |                     | the field label                                   |
+| `label`           | function        | @?      | yes              |             |                     | a text to help fill the form field                |
+
+### error-messages
+
+| Attribute         | Type            | Binding | One-time binding | Values      | Default                                   | Description                 |
```

Comment:
It's not really a tag attribute, so... some columns should not be there like "Binding", "One-time binding"...
Maybe it's could be sufficient to only show the default object and explaining that it can be overridden.

Reply:
I think it could be replaced with a configuration section like for pagination component.
An exemple of provider configuration with default message could be interesting

Reply:
Yeah, the binding is for the object containing the messages, I was was not sure on how to present it. I will take a look at the pagination section to try to improve that.